index.bs

```diff
@@ -1897,15 +1897,18 @@ an |ownership type|, a |serialization internal map| and a |realm|:
 
           1. Set |serialized|["<code>nodeType</code>"] to [=Get=](|value|, "nodeType").
 
-          1. Set |serialized|["<code>nodeValue</code>"] to [=Get=](|value|, "nodeValue")
+          1. Set |node value| to [=Get=](|value|, "nodeValue").
+
+          1. If |node value| is not undefined set
```

Comment:
It can't be `undefined`, but can be `null`: https://dom.spec.whatwg.org/#dom-node-nodevalue

Reply:
Ups, not sure why I used `undefined` here. Thanks for catching that.